lib/sql/util.go

```diff
@@ -0,0 +1,14 @@
+package sql
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/artie-labs/transfer/lib/stringutil"
+)
+
+func QuoteLiteral(value string) string {
```

Comment:
This is https://github.com/artie-labs/transfer/pull/538/files#diff-22f228280edef1bb7b7e56fe7a65982f4ca45126af422ad68da566b9e1fb7611L29 with `noQuotes: false`.